src/utils/actions.ts

```diff
@@ -12,12 +12,14 @@ enum ACTION_NAME {
     PAGE_TOP = "pageTop",
     PAGE_BOTTOM = "pageBottom",
     RELOAD = "reload",
+    COPY_LOG_EVENT = "copyLogEvent",
 }
 
 interface EditorAction {
     actionName: Nullable<ACTION_NAME>;
     label: string;
     keyBindings: monaco.KeyCode[];
+    contextMenuGroupId?: string;
```

Comment:
Instead of using `?` (potentially `undefined`), can we make the property `Nullable<string>` instead?